main.tf

```diff
@@ -131,6 +156,27 @@ data "aws_iam_policy_document" "resource_full_access" {
 
     actions = ["ecr:*"]
   }
+
+  dynamic "statement" {
```

Comment:
why include this block in the full access document? shouldn't it only be in the read only document?